toml.md

```diff
@@ -553,6 +553,15 @@ time with a space character (as permitted by RFC 3339 section 5.6).
 odt4 = 1979-05-27 07:32:00Z
 ```
 
+One exception to RFC 3339 is permitted: seconds may be omitted if the instant
+occurs at a zero-second mark. In such a case, the offset immediately follows
+the minutes.
```

Comment:
I do state that this can be done "if the instance occurs at a zero-second mark". Do I need to be more explicit than that? We don't introduce anything like a minute-accurate timestamp data type anywhere, so I'm inclined to just leave this be.

I could be warmed up to @arp242's suggestion. We could put "(in which case `:00` will be assumed)" just before the offset remark.

Thoughts?

Reply:
I like @arp242 's proposal too, that is, "if the instant occurs at a zero-second mark" should be replaced with "in which case `:00` will be assumed". There is no need to have both, it would be redundant.

Reply:
It's done. Check the latest commit. I updated `toml.md` in all three places to read "seconds may be omitted, in which case `:00` will be assumed."